lib/mochaReporter/lib/reportStatus/lib/getRepoFromPackageJson.js

```
@@ -0,0 +1,29 @@
+"use strict";
+
+const path = require('path');
+const log = require('@studyportals/node-log');
+
+/**
+ * Gets the repo of the current project, executing the tests.
+ * @return {string | null}
+ */
+module.exports = () => {
+	try{
+		const packageJsonPath = path.resolve(process.cwd(), "package.json");
+		const packageJson = require(packageJsonPath);
+
+		let repository = packageJson.repository;
+		repository = repository.url;
+		repository = repository.split("/");
+		repository = repository.slice(-1);
+		repository = repository[0];
+		repository = repository.replace(".git", "");
+		return repository;
+	}
+	catch(ex){
+
+		log.error("Could not determine repo in " + process.cwd());
+		log.error(ex);
```

I am not sure if my logging package handles Objects well. I would just re-throw the error and let the implementation decide what to do with it.